electron-app/src/components/settings/api-keys/ExchangeSettings.vue

```diff
@@ -68,6 +68,12 @@
             :type="showKey ? 'text' : 'password'"
             @click:append="showKey = !showKey"
           ></v-text-field>
+          <v-select
+            v-model="selectedKrakenAccountType"
+            class=""
+            :items="krakenAccountTypes"
```

Comment:
you can add `@change` here pointing to the change function instead of `@watch`

Reply:
done